app/models/champs/email_champ.rb

```diff
@@ -1,2 +1,3 @@
 class Champs::EmailChamp < Champs::TextChamp
+  validates :value, format: { with: Rails.application.config.devise.email_regexp }
```

Comment:
```suggestion
  validates :value, format: { with: Rails.application.config.devise.email_regexp }, if: :validate_champ_value?
```

Il faut valider que lors des validations explicites, tout comme pour les autres validations des champs

Reply:
aussi, d'habitude on passe par `Devise.email_regexp`